pages/templates/pages/group_page.html

```diff
@@ -59,20 +59,24 @@ <h1 class="fw7 mt0">{{ group.name }}</h1>
         {% if group.rep_email %}
           <h3 class="mt0 mb0">Group Representative</h3>
           <div><strong>{{ group.rep_first_name }} {{ group.rep_last_name }}</strong></div>
-          <div><a href="mailto:{{ group.rep_email }}?Subject={{group.name|urlencode}}">{{ group.rep_email }}</a></div>
+          <div>
+            <a href="mailto:{% firstof group.group_contact_email group.rep_email %}?Subject={{group.name|urlencode}}">
```

Comment:
We should only display public contact email here, not one used for BSD authentication.